telegram/ext/conversationhandler.py

```diff
@@ -331,8 +338,9 @@ def handle_update(self, update, dispatcher, check_result):
         if self.conversation_timeout and new_state != self.END:
             self.timeout_jobs[conversation_key] = dispatcher.job_queue.run_once(
                 self._trigger_timeout, self.conversation_timeout,
-                context={'current_conversation': conversation_key, 'update': update,
-                         'dispatcher': dispatcher})
+                context=ConversationTimeoutContext(conversation_key=conversation_key,
```

Comment:
Why not just `_ConversationTimeoutContext(conversation_key, update, dispatcher)`?
(i'm just nitpicking after our long discussion what's an unneeded overhead)